samples/peripheral/802154_phy_test/README.rst

```diff
@@ -0,0 +1,357 @@
+     - custom lclk 10 1
+   * - Set GPIO pin value
+     - custom lsetgpio <pin> <value>
+     - <pin> - GPIO pin number, 0-N, where N depends on used SoC; <value> - high when 1, low when 0
+     - CMD will set value of the selected GPIO pin if it’s an out pin
+     - custom lsetgpio 29 0
+   * - Get GPIO pin value
+     - custom lgetgpio <pin>
+     - <pin> - GPIO pin number, 0-N, where N depends on used SoC
+     - CMD will reconfigure selected GPIO pin to INPUT mode and read its value
+     - custom lgetgpio 29
+   * - Set DC/DC mode
+     - custom lsetdcdc <value>
+     - <value> - enabled when 1, disabled when 0
+     - CMD will disable/enable DC/DC mode (no effect on unsupported boards)
+     - custom lsetdcdc 1
+   * - Get DC/DC mode
+     - custom lgetdcdc
+     - 
+     - Gets the DC/DC mode of CMD device, always 0 for unsupported boards
+     - custom lgetdcdc
+   * - Set ICACHE configuration
+     - custom lseticache <value>
+     - <value> - enabled when 1, disabled when 0
+     - CMD will disable/enable ICACHE
+     - custom lseticache 1
+   * - Read SoC temperature
+     - custom lgettemp
+     - 
+     - CMD will print SoC temperature in the following format: <.%02>
+     - custom lgettemp
+   * - Transition radio to sleep mode
+     - custom lsleep
+     - 
+     - CMD will put radio into sleep mode
+     - custom lsleep
+   * - Transition radio to receive mode
+     - custom lreceive
+     - 
+     - CMD will put radio in receive mode
+     - custom lreceive
+   * - Reboot the device
+     - custom lreboot
+     - 
+     - Reboots the device
+     - custom lreboot
+
+
+   
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/peripheral/802154_phy_test`
+
+.. include:: /includes/build_and_run.txt
+
+.. note::
+   On the |nRF5340DKnoref|, the IEEE 802.15.4 PHY Test Tool is a standalone network sample that does not require any counterpart application sample.
+   However, you must still program the application core to boot up the network core.
+   You can use any sample for this, for example :ref:`nrf5340_empty_app_core`.
+   The :ref:`nrf5340_empty_app_core` is built and programmed automatically by default.
+   If you want to program another sample for the application core, unset the :option:'CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE' option.
+
+.. _802154_phy_test_testing:
+
+Using the tool
+==============
+
+After programming the supported development kit of your choice with the sample, you can verify that it functions as expected and begin performing tests.
+
+.. _802154_phy_test_testing_check:
+
+Checking if the sample works as expected
+----------------------------------------
+
+1. Connect the development kit to the computer using a USB cable. Use the DK's programmer USB port, not the nRF USB port located near the SoC.
```

Comment:
I think it would be better to use ports names like J2, J3. These names are printed on the PCB